### PR TITLE
Add Spot ORM

### PR DESCRIPTION
**Pull request by urisavka**
Adding [Spot2](https://github.com/vlucas/spot2) ORM support.

### CODE REVIEW

**Comment by fzaninotto** [comment]
Thanks!

File: src/Faker/ORM/Spot/ColumnTypeGuesser.php [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+                return function () use ($generator, $size) {
+                    return $generator->randomNumber($size + 2) / 100;
+                };
+            case 'smallint':
+                return function () {
+                    return mt_rand(0, 65535);
+                };
+            case 'integer':
+                return function () {
+                    return mt_rand(0, intval('2147483647'));
```
Comment (TimWolla) [inline src/Faker/ORM/Spot/ColumnTypeGuesser.php]: Why don't you use `$generator->numberBetween()` here?

File: src/Faker/ORM/Spot/ColumnTypeGuesser.php [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+                };
+            case 'bigint':
+                return function () {
+                    return mt_rand(0, intval('18446744073709551615'));
+                };
+            case 'float':
+                return function () {
+                    return mt_rand(0, intval('4294967295'))/mt_rand(1, intval('4294967295'));
```
Comment (TimWolla) [inline src/Faker/ORM/Spot/ColumnTypeGuesser.php]: `$generator->randomFloat()`. In fact I am not entirely sure whether your current method generates a uniform distribution (I suspect not).

File: src/Faker/ORM/Spot/ColumnTypeGuesser.php (hunk 1 above)
Comment (urisavka) [inline src/Faker/ORM/Spot/ColumnTypeGuesser.php]: Copied it from Doctrine column type guesser. Will fix both today. 

I think ``$generator->randomNumber(10);`` is even more correct solution here...

File: src/Faker/ORM/Spot/ColumnTypeGuesser.php (hunk 2 above)
Comment (urisavka) [inline src/Faker/ORM/Spot/ColumnTypeGuesser.php]: Same as above. Thanks for noticing!

**Pull request merged**